test/com/google/javascript/jscomp/TypeCheckTest.java

```diff
@@ -4043,7 +4042,7 @@ public void testInterfaceExtends() {
   }
 
   public void testDontCrashOnDupPropDefinition() {
-    testTypes(Joiner.on('\n').join(
+    testTypes(LINE_JOINER.join(
```

Comment:
https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/rhino/testing/BaseJSTypeTestCase.java#L58
already has a LINE_JOINER defined, so the one in `TypeCheckTest` was shadowing that and seems unnecessary.